### ISSUE TITLE
(Img2Img) RuntimeError: The size of tensor a (128) must match the size of tensor b (64) at non-singleton dimension 3

### ISSUE THREAD

**Issue by Lyntai** [altryne/sd-webui-colab#13]
Img2Img
```
File "scripts/webui.py", line 1194, in sample
    xi = x0 + noise
RuntimeError: The size of tensor a (128) must match the size of tensor b (64) at non-singleton dimension 3
```

**Comment by Lop010** [comment]
I am also having this issue

**Comment by altryne** [comment]
Yeah this was fixed on main branch, I need to wait until it's stable and update the colab fork.

**Comment by altryne** [comment]
@Lop010 please try the latest version, and let me know?

**Comment by Lop010** [comment]
Hey tried it just now and still having the same issue

**Comment by altryne** [comment]
can you uncheck this and try? 
![CleanShot 2022-09-03 at 15 15 16@2x](https://user-images.githubusercontent.com/463317/188287872-ef17ba1b-0c30-4447-b4b0-3a62d21b6d85.jpg)

**Comment by scottincrypto** [comment]
> can you uncheck this and try? ![CleanShot 2022-09-03 at 15 15 16@2x](https://user-images.githubusercontent.com/463317/188287872-ef17ba1b-0c30-4447-b4b0-3a62d21b6d85.jpg)

Hey, just tried this - same error
